examples/alerts/alerts.md

```diff
@@ -106,7 +106,7 @@ rules:
     sum by (job) (rate(thanos_rule_evaluation_with_warnings_total{job=~"thanos-rule.*"}[5m])) > 0
   for: 15m
   labels:
-    severity: warning
+    severity: info
```

Comment:
Downgraded to `info` because it won't affect anything operational, Thanos ruler will continue to operate regardlessly.